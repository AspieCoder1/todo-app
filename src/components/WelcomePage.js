import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import styles from '../styles/components/welcomePage.module.scss';
import Header from './Header';

export default class WelcomePage extends Component {
	render() {
		return (
			<div className={styles.page}>
				<Header />
				<div className={styles.content}>
					<div>
						<h1 className={styles.title}>Time to take back control</h1>
						<div className={styles.buttonGroup}>
							<Link className={styles.button} to="/login">
								Login
							</Link>
							<Link className={styles.button} to="/register">
								Register
							</Link>
						</div>
					</div>
				</div>
			</div>
		);
	}
}
