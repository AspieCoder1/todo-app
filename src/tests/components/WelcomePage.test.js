import React from 'react';
import { shallow } from 'enzyme';

import WelcomePage from '../../components/WelcomePage';

it('should render WelcomePage correctly', () => {
	const wrapper = shallow(<WelcomePage />);
	expect(wrapper).toMatchSnapshot();
});
