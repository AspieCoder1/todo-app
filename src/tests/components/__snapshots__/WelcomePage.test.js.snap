// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render WelcomePage correctly 1`] = `
<div
  className="page"
>
  <_default />
  <div
    className="content"
  >
    <div>
      <h1
        className="title"
      >
        Time to take back control
      </h1>
      <div
        className="buttonGroup"
      >
        <Link
          className="button"
          replace={false}
          to="/login"
        >
          Login
        </Link>
        <Link
          className="button"
          replace={false}
          to="/register"
        >
          Register
        </Link>
      </div>
    </div>
  </div>
</div>
`;
